docker/ubuntu18/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:18.04
+LABEL maintainer="Shaun Jackman <sjackman@gmail.com>"
```

Comment:
Should we move the email adress to some SFC email one day? This does not need to be done in this PR, it's something for later.

Reply:
I think it would be good to do in this PR.

Reply:
That's fine by me. Which e-mail address do you suggest?

Reply:
Easier, I'll remove this line altogether.